test/test_mu_law.py

```diff
@@ -0,0 +1,35 @@
+import tensorflow as tf
```

Comment:
Should be:

```python
import numpy as np
import tensorflow as tf

from wavenet import WaveNet
```

Unless we treat wavenet as a separate package, in which case the empty line should be removed.